src/components/SQForm/SQForm.tsx

```diff
@@ -34,17 +64,25 @@ function SQForm({
   // HACK: This is a workaround for: https://github.com/mui-org/material-ui-pickers/issues/2112
   // Remove this reset handler when the issue is fixed.
   const handleReset = () => {
-    document && document.activeElement && document.activeElement.blur();
+    document &&
+      document.activeElement &&
+      (document.activeElement as HTMLElement).blur();
```

Comment:
[tiny nit] Not sure if prettier or what is going on here, but the indentation is funny.